Client/Frontend/Browser/TabLocationView.swift

```diff
@@ -234,10 +280,58 @@ class TabLocationView: UIView {
             attributedString.addAttribute(UIAccessibilitySpeechAttributePitch, value: NSNumber(double: TabLocationViewUX.BaseURLPitch), range: nsRange)
             attributedString.pitchSubstring(baseDomain, withPitch: TabLocationViewUX.HostPitch)
             urlTextField.attributedText = attributedString
+ 
         } else {
             // If we're unable to highlight the domain, just use the URL as is.
             urlTextField.text = url?.absoluteString
         }
+        */
+        
+        // Cliqz: disply only the domain name in the url TextField if possible
+        var urlText = ""
+        if let baseDomain = url?.baseDomain() {
+            urlText = baseDomain
+        } else if let absoluteString = url?.absoluteString {
+            urlText = absoluteString
+        } else {
+            urlTextField.text = ""
+            urlTextField.textAlignment = .Left
+            urlTextFieldWidth = getMaxUrlTextFieldWidth()
+            return
+        }
+        let attributedString = NSMutableAttributedString(string: urlText)
+        let nsRange = NSMakeRange(0, urlText.characters.count)
+        
+        attributedString.addAttribute(NSForegroundColorAttributeName, value: hostFontColor, range: nsRange)
+        attributedString.addAttribute(UIAccessibilitySpeechAttributePitch, value: NSNumber(double: TabLocationViewUX.HostPitch), range: nsRange)
+        
+        urlTextField.attributedText = attributedString
+        urlTextField.textAlignment = .Center
+        urlTextFieldWidth = attributedString.size().width + 25
```

Comment:
it is a margin as `attributedString.size().width` does not accurately calculate the width